tests/test_config.py

```diff
@@ -736,20 +736,25 @@ async def test_check_ha_config_file_wrong(mock_check, hass):
     assert await config_util.async_check_ha_config_file(hass) == "bad"
 
 
-@patch("homeassistant.config.os.path.isfile", mock.Mock(return_value=True))
-async def test_async_hass_config_yaml_merge(merge_log_err, hass):
+@pytest.mark.parametrize(
+    "yaml_config",
+    [
+        {
+            config_util.CONF_CORE: {
+                config_util.CONF_PACKAGES: {
+                    "pack_dict": {"input_boolean": {"ib1": None}}
+                }
+            },
+            "input_boolean": {"ib2": None},
+            "light": {"platform": "test"},
+        }
+    ],
+)
+async def test_async_hass_config_yaml_merge(
+    merge_log_err, hass: HomeAssistant, mock_yaml_config: None
+) -> None:
```

Comment:
I wonder if there are more places where `mock_yaml_config` could be used already.
Maybe `test_bootstrap.py`

Or maybe we should be patching "homeassistant.config.async_hass_config_yaml" instead?

Reply:
I added some applications to the PR.